iofog_version_controller/src/org/eclipse/iofog_version_controller/command_line/util/CommandShellExecutor.java

```diff
@@ -72,4 +75,9 @@
 		}
 		return result;
 	}
+
+	private static boolean isWindows() {
+		String osName = getSystemProperty("os.name");
+		return osName != null && osName.startsWith("Windows");
+	}
```

Comment:
can we just use SystemUtils.IS_OS_WINDOWS here?

Reply:
not here, it's separate iofog_version_controller jar which has no dependencies, so just using the same simplified code from SystemUtils